source/includes/aggregation.rb

```diff
@@ -0,0 +1,61 @@
+require 'bundler/inline'
+gemfile do
+  source 'https://rubygems.org'
+  gem 'mongo'
+end
+
+uri = '<connection string URI>'
+
+Mongo::Client.new(uri) do |client|
+  #start-aggregation
+  database = client.use('sample_restaurants')
+  restaurants_collection = database[:restaurants]
+    
+  pipeline = [
+    { '$match' => { 'cuisine' => 'Bakery' } },
+    { '$group' => {
+      '_id' => '$borough',
+      'count' => { '$sum' => 1 }
+      }
+    }
```

Comment:
Indenting nested hashes like this should indent their contents relative to the key, and not the outer bracket. It's not syntactically wrong to do otherwise, but adding the indentation does make it easier to see how things are nested.

```suggestion
    { '$group' => {
        '_id' => '$borough',
        'count' => { '$sum' => 1 }
      }
    }
```